src/config.rs

```diff
@@ -680,6 +708,40 @@ fn config_from_env() -> Result<EnvConfig> {
         None
     };
 
+    // ======= Buildless =======
+    let buildless = if
+        env::var("SCCACHE_BUILDLESS").is_ok() ||
+        env::var("BUILDLESS_APIKEY").is_ok() {
+        // read kill switch
+        let enable_var = env::var("SCCACHE_BUILDLESS")
```

Comment:
buildless is detected last for configuration, in order to avoid failing tests for other providers which expect to fail if config is incomplete